Data/Migrations/20210323135013_itemviewmodel.cs

```diff
@@ -0,0 +1,223 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace HowManyCaloriesIn.Data.Migrations
+{
+    public partial class itemviewmodel : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_tblNutrients_tblItem_ItemID",
+                table: "tblNutrients");
+
+            migrationBuilder.DropPrimaryKey(
+                name: "PK_tblItem",
+                table: "tblItem");
+
+            migrationBuilder.RenameTable(
+                name: "tblItem",
+                newName: "Item");
+
+            migrationBuilder.RenameColumn(
+                name: "ItemID",
+                table: "tblNutrients",
+                newName: "ItemId");
+
+            migrationBuilder.RenameColumn(
+                name: "ID",
+                table: "tblNutrients",
+                newName: "Id");
+
+            migrationBuilder.RenameIndex(
+                name: "IX_tblNutrients_ItemID",
+                table: "tblNutrients",
+                newName: "IX_tblNutrients_ItemId");
+
+            migrationBuilder.RenameColumn(
+                name: "ID",
+                table: "Item",
+                newName: "Id");
+
+            migrationBuilder.AlterColumn<double>(
+                name: "Sugars",
+                table: "tblNutrients",
+                nullable: false,
+                oldClrType: typeof(int),
+                oldType: "int");
+
+            migrationBuilder.AlterColumn<double>(
+                name: "SaturatedFat",
+                table: "tblNutrients",
+                nullable: false,
+                oldClrType: typeof(int),
+                oldType: "int");
+
+            migrationBuilder.AlterColumn<double>(
+                name: "Salt",
+                table: "tblNutrients",
+                nullable: false,
+                oldClrType: typeof(int),
+                oldType: "int");
+
+            migrationBuilder.AlterColumn<double>(
+                name: "Fat",
+                table: "tblNutrients",
+                nullable: false,
+                oldClrType: typeof(int),
+                oldType: "int");
+
+            migrationBuilder.AlterColumn<string>(
+                name: "Name",
+                table: "AspNetUserTokens",
+                maxLength: 128,
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(450)");
+
+            migrationBuilder.AlterColumn<string>(
+                name: "LoginProvider",
+                table: "AspNetUserTokens",
+                maxLength: 128,
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(450)");
+
```

Comment:
I had to remove the default user column changes as it was causing constraint issues. Not sure if you intended to make these changes.